torch/testing/_internal/common_utils.py

```diff
@@ -63,7 +63,7 @@
 if sys.platform == 'win32':
     FILE_SCHEMA = "file:///"
 
-IS_IN_CI = bool(os.environ.get('IN_CI'))
+IS_IN_CI = os.getenv('IN_CI') == '1'
```

Comment:
Do we want to explicitly add a default (currently it's None so that'd be fine too)

Reply:
i think we are fine (others are doing the same and seems to work quite well both locally and on CI)